alzueta.yaml

```diff
@@ -0,0 +1,1849 @@
+    - {P: 1.000e+00 atm, A: 3.65176e+26, b: -5.65302e+00, Ea: 2.95891e+03}
+    - {P: 1.000e+01 atm, A: 4.51325e+23, b: -4.50139e+00, Ea: 2.58051e+03}
+    - {P: 1.000e+02 atm, A: 2.56240e+20, b: -3.31075e+00, Ea: 2.07400e+03}
+    - {P: 1.000e+03 atm, A: 8.99670e+16, b: -2.12230e+00, Ea: 1.37012e+03}
+    - {P: 1.000e+04 atm, A: 5.28067e+13, b: -1.06166e+00, Ea: 5.64223e+02}
+  - name: N2
+    eps: {A: 1.46848e+01, b: -3.00962e-01, Ea: 1.65146e+02}
+  - name: O2
+    eps: {A: 1.53608e+00, b: -3.90857e-02, Ea: 4.43752e+00}
+  - name: NH3
+    eps: {A: 1.64196e+01, b: -7.29636e-02, Ea: 3.66099e+02}
+  - name: H2O
+    eps: {A: 2.17658e+01, b: -1.14715e-01, Ea: 4.19216e+02}
+- equation: HNO (+M) <=> H + NO (+M)  # Reaction 167
+  type: linear-Burke
+  colliders:
+  - name: M # Ar is reference collider
+    eps: {A: 1, b: 0, Ea: 0}
+    rate-constants:
+    - {P: 0.1 atm, A: 2.0121e+19, b: -3.021, Ea: 47792.0}
+    - {P: 1 atm, A: 1.8259e+20, b: -3.008, Ea: 47880.0}
+    - {P: 10 atm, A: 1.2762e+21, b: -2.959, Ea: 48100.0}
+    - {P: 100 atm, A: 5.6445e+21, b: -2.855, Ea: 48459.0}
+    - {P: 1000 atm, A: 9.7111e+21, b: -2.642, Ea: 48940.0}
+  - name: N2
+    eps: {A: 7.56771e+00, b: -1.96878e-01, Ea: 2.53168e+02}
+  - name: H2O
+    eps: {A: 3.35451e+00, b: 1.33352e-01, Ea: 6.64633e+01}
+
+common_reactions:
+- equation: H + O2 <=> O + OH  # Reaction 1
+  rate-constant: {A: 1.0e+14, b: 0.0, Ea: 1.5286e+04}
+- equation: O + H2 <=> OH + H  # Reaction 2
+  duplicate: true
+  rate-constant: {A: 3.8e+12, b: 0.0, Ea: 7948.0}
+- equation: O + H2 <=> OH + H  # Reaction 3
+  duplicate: true
+  rate-constant: {A: 8.8e+14, b: 0.0, Ea: 1.9175e+04}
+- equation: OH + H2 <=> H + H2O  # Reaction 4
+  rate-constant: {A: 2.2e+08, b: 1.51, Ea: 3430.0}
+- equation: OH + OH <=> O + H2O  # Reaction 5
+  duplicate: true
+  rate-constant: {A: 2.0e+07, b: 1.651, Ea: 631.0}
+- equation: OH + OH <=> O + H2O  # Reaction 6
+  duplicate: true
+  rate-constant: {A: 2.6e+11, b: -0.057, Ea: -827.0}
+- equation: H2 + M <=> H + H + M  # Reaction 7
+  type: three-body
+  rate-constant: {A: 4.6e+19, b: -1.4, Ea: 1.0438e+05}
+  efficiencies: {H2: 2.5, H2O: 12.0, AR: 0.0, O2: 0.0}
+- equation: H2 + O2 <=> H + H + O2  # Reaction 8
+  duplicate: true
+  rate-constant: {A: 4.6e+19, b: -1.4, Ea: 1.0438e+05}
+- equation: H2 + AR <=> H + H + AR  # Reaction 9
+  rate-constant: {A: 5.8e+18, b: -1.1, Ea: 1.0438e+05}
+- equation: H + O + M <=> OH + M  # Reaction 10
+  type: three-body
+  rate-constant: {A: 4.7e+18, b: -1.0, Ea: 0.0}
+  efficiencies: {H2: 2.5, H2O: 12.0, AR: 0.75, CO: 1.9, CO2: 3.8, O2: 0.0}
+- equation: H + O + O2 <=> OH + O2  # Reaction 11
+  duplicate: true
+  rate-constant: {A: 4.7e+18, b: -1.0, Ea: 0.0}
+- equation: O + O + M <=> O2 + M  # Reaction 12
+  type: three-body
+  rate-constant: {A: 1.9e+13, b: 0.0, Ea: -1788.0}
+  efficiencies: {H2: 2.5, H2O: 12.0, AR: 0.0, CO: 1.9, CO2: 3.8}
+- equation: H2O + O2 <=> H + OH + O2  # Reaction 14
+  duplicate: true
```

Comment:
I think you should either delete or correct all the `Reaction NN` comments, as they don't currently correspond to the reaction numbers when importing the mechanism.